src/master-playlist-controller.js

```diff
@@ -553,7 +553,7 @@ export class MasterPlaylistController extends videojs.EventTarget {
       const codecs = this.getCodecsOrExclude_();
 
       // no codecs means that the playlist was excluded
-      if (!codecs || !this.sourceUpdater_.canChangeType()) {
+      if (!codecs) {
```

Comment:
We do not want to check `canChangeType` here, as we do it in `getCodecsOrExclude` and blacklist if the main part of the codec was changed, but sourcebuffers can not actually changetype.